client/components/Form/FormGroup/FormGroup.module.scss

```diff
@@ -14,4 +14,10 @@
     width: 100%;
     display: block;
   }
+  input[type="text"]:focus,
```

Comment:
This PR should only be related to centering the icon inside the `<PasswordReveal>` component. Let's revert this code.